interview-questions.md

```diff
@@ -4,51 +4,51 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What is object destructuring?
 
-  Your answer:
+  Your answer: object destructuring is when you define an object and how it will be referenced later. this is done to allow for objects to be called using a shorter notation
 
-  Researched answer:
+  Researched answer: destructuring is a way of assigning variables to objects.
 
 
 
 2. What are the three arguments that .map() can accept?
 
-  Your answer:
+  Your answer: you can use .map() to pass an argument through an array and it will create a new array. it is often used to perfrom math opperations on integers. 
```

Comment:
true! but .map takes in three arguments as well: value, index, and the array that it will be iterating through :)